src/javascript/class-scanner.ts

```diff
@@ -385,41 +388,27 @@ class PrototypeMemberFinder implements Visitor {
     }
 
     if (babel.isFunctionExpression(node.right)) {
-      const method = this._createMethodFromExpression(leftProperty.name,
+      const member = this._createMethodFromExpression(leftProperty.name,
```

Comment:
Super minor, but `method` and `prop` are clearer to me than the generic `member`, when we do know what kind of thing it is.